cfme/tests/webui/test_general_ui.py

```diff
@@ -288,6 +289,50 @@ def test_add_button_on_provider_all_page(
     assert displayed_view.is_displayed
 
 
+@pytest.mark.tier(1)
+@pytest.mark.meta(automates=[1475553])
+def test_tls_openssl_verify_mode(temp_appliance_preconfig, request):
+    """
+    Polarion:
+        assignee: pvala
+        casecomponent: WebUI
+        caseimportance: medium
+        initialEstimate: 1/8h
+        testSteps:
+            1. Check if key `openssl_verify_mode` is present in the advanced configuration.
+            2. Navigate to Configuration and toggle `Start TLS Automatically`
+                of Outgoing SMTP E-mail Server.
+            3. Again check for the presence of `openssl_verify_mode` and check its value.
+        expectedResults:
+            1. Key must be absent.
+            2.
+            3. Key must be present and value must be None.
+
+    Bugzilla:
+        1475553
+    """
+    # take a fresh appliance where smtp has never been touched
+    appliance = temp_appliance_preconfig
+    # 1
+    assert "openssl_verify_mode" not in appliance.advanced_settings["smtp"]
+    # 2
+    view = navigate_to(appliance.server, "Server")
+
+    # start_tls will be set to False since it's default value is True,
+    # although value of start_tls doesn't really affect the value of openssl_verify_mode
+    appliance.server.settings.update_smtp_server(
+        {"start_tls": not view.smtp_server.start_tls.read()}
+    )
+
+    # 3
+    wait_for(
+        lambda: "openssl_verify_mode" in appliance.advanced_settings["smtp"],
+        timeout=50,
+        delay=2,
+    )
+    assert appliance.advanced_settings["smtp"]["openssl_verify_mode"] == "none"
```

Comment:
**Question** So regardless of the value of `start_tls`, `open_verify_mode` should be set to none correct? I think it is worth changing `start_tls` back to what the original value was and verifying that `start_tls` was changed back and `openssl_verify_mode` is still none. Thoughts?

Reply:
Correct, it will always be none, unless you change the value of `SSL Verify Mode`.

You need to either change `SSL Verify Mode` or `Start TLS Automatically` to introduce `openssl_verify_mode` into the advanced_config YAML.

>Thoughts?

Um, maybe. I guess there's no harm in adding those few more steps.

Reply:
Thanks @valaparthvi , I think it would make the test more complete.